src/annotations/AnnotationManager.php

```diff
@@ -299,6 +306,21 @@ protected function getAnnotations($class_name, $member_type = self::MEMBER_CLASS
                 }
             }
 
+            if ($this->_traitsSupported && !isset($file->data[$key]) && $member_name !== null) {
+                $traitAnnotations = array();
+
+                if (isset($file) && isset($file->traitMethodOverrides[$class_name][$member_name])) {
```

Comment:
I've just checked, that `AnnotationParser` ensures that it's an empty array in case hence no notice.